pkg/mcs/resourcemanager/server/manager.go

```diff
@@ -129,7 +129,9 @@ func (m *Manager) Init(ctx context.Context) error {
 		return err
 	}
 	// Load resource group meta info from storage.
+	m.Lock()
 	m.groups = make(map[string]*ResourceGroup)
+	m.Unlock()
```

Comment:
Where will it be called concurrently?

Reply:
I think because there is no happens-before guarantee for different goroutines, maybe meet memory reordering.